perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/VosManagerMethod.java

```diff
@@ -428,16 +428,15 @@ public Void call(ApiCaller ac, Deserializer parms)
 		public List<User> call(ApiCaller ac, Deserializer parms) throws PerunException {
 			if(parms.contains("role")) {
 				String roleName = parms.readString("role");
-				Role role;
-				try {
-					role = Role.valueOf(roleName);
-				} catch (IllegalArgumentException ex) {
-					throw new RpcException(RpcException.Type.WRONG_PARAMETER, "wrong parameter in role, not exists role with this name " + roleName);
+				if (roleName == null) {
+					throw new RpcException(RpcException.Type.WRONG_PARAMETER, "wrong parameter in role, cannot be null.");
```

Comment:
Again please fix all occurrences of this exception message.

Reply:
Done